### PR TITLE
Added uint[2,3,4] to simd, bridged to vector_uintN.

### PR DESCRIPTION
**Pull request by stephentyrone**
There are a few ModelIO APIs that cannot currently be imported because these types aren't bridged to Swift.  This pull adds uint2, uint3, uint4 types to the simd module, bridged to vector_uintN in Obj-C, allowing those APIs to be imported.

Before merging this pull request to apple/swift repository:
- [x] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

 **Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| OS X platform | @swift-ci Please test OS X platform |
| Linux platform | @swift-ci Please test Linux platform |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

**Comment by stephentyrone** [comment]
@swift-ci Please test OS X platform

**Comment by gribozavr** [comment]
@swift-ci Please test

File: test/Inputs/clang-importer-sdk/swift-modules-without-ns/simd.swift [hunk 1]
```diff
@@ -2,3 +2,4 @@
 @_alignment(16) public struct float4 { public var x, y, z, w: Float }
 @_alignment(16) public struct double2 { public var x, y: Double }
 @_alignment(16) public struct int3 { public var x, y, z: Int32 }
+@_alignment(16) public struct uint4 { public var x, y, z, w: UInt32 }
```
Comment (jckarter) [inline test/Inputs/clang-importer-sdk/swift-modules-without-ns/simd.swift]: Missing newline

File: test/Inputs/clang-importer-sdk/swift-modules/simd.swift [hunk 2]
```diff
@@ -2,3 +2,4 @@
 @_alignment(16) public struct float4 { public var x, y, z, w: Float }
 @_alignment(16) public struct double2 { public var x, y: Double }
 @_alignment(16) public struct int3 { public var x, y, z: Int32 }
+@_alignment(16) public struct uint4 { public var x, y, z, w: UInt32 }
```
Comment (jckarter) [inline test/Inputs/clang-importer-sdk/swift-modules/simd.swift]: Missing newline

**Comment by jckarter** [comment]
LGTM, modulo some missing newlines.

**Comment by stephentyrone** [comment]
Updated with newlines that @jckarter requested.  OK to merge?

**Comment by jckarter** [comment]
Go for it.

**Comment by lattner** [comment]
Nice, please update the changelog.md file @stephentyrone !

**Pull request merged**